index.html

```diff
@@ -11,6 +11,22 @@ <h1>Monthly Lollipops</h1>
     </div>
     <script>
       // Monthly Lollipops JS
+
```

Comment:
Hrmm it's generally frowned upon to have scripts on the html pages. They're usually put in separate js files and then referenced at the top of the main html page.
Do you know if this is a D3 thing?

Reply:
Not a D3 thing, I think. More about how it can take longer to load JS from separate files, especially if the code isn't long.
I'm okay with using the app.js file in your PR.